lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -199,8 +199,12 @@ def add_jump_to_view(node)
           .downcase
 
         view_uris = Dir.glob("#{@client.rails_root}/app/views/#{controller_name}/#{action_name}*").map! do |path|
+          # it's possible we could have a directory with the same name as the action, so we need to skip those
+          next if FileTest.directory?(path)
```

Comment:
I've never seen `FileTest` used before. Is this equivalent to `File.directory?` and is there a reason to prefer this one?